README.md

```diff
@@ -89,6 +92,8 @@ cat atlas.clusters.2.0.GRCm38.96.bed | sed -E 's/^([0-9]+|[XY])/chr\1/' | sed -E
 # Example for human
 sed -E 's/^([0-9]+|[XYM])/chr\1/' atlas.clusters.2.0.GRCh38.96.bed > atlas.clusters.2.0.GRCh38.96.wchr.bed
 ```
+> Please be mindful of the `biotype_key` key in the configuration file, which should be set according to the annotation type provided to the pipeline.
```

Comment:
Great point! Could be moved to line 83 (after first sentence of "Annotations and poly(A) sites") though. Like   
"[...] and gene annotations (e.g. from [ensembl][ensembl]) for your organism and specify their paths in the `config.yaml`. *Please be mindful of the `biotype_key` key in the configuration file, which should be set according to the annotation type provided to the pipeline (e.g. "transcript_biotype" for ensembl gtf files, "transcript_type" for gencode gtf files).*"

Reply:
Done!